admin/tools/docker/replication/deployment/ncsa/env_svc.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+# This file is part of {{ cookiecutter.package_name }}.
```

Comment:
`{{ cookiecutter.package_name }}` -> qserv (and four other cases below).

Reply:
Good catch! Fixed in this and in other scripts.